CommonAssemblyInfo.cs

```diff
@@ -10,7 +10,7 @@
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("Git Extensions")]
 [assembly: AssemblyProduct("Git Extensions")]
-[assembly: AssemblyCopyright("Copyright © 2008-2019 Git Extensions Team")]
+[assembly: AssemblyCopyright("Copyright © 2008-2023 Git Extensions Team")]
```

Comment:
This is unrelated

Reply:
Remains from changing the format, can revert